tests/test_smartcar.py

```diff
@@ -45,14 +45,60 @@ def test_is_expired(self):
         self.assertTrue(smartcar.is_expired(access['expiration']))
 
     def test_get_auth_url(self):
+        client = smartcar.AuthClient(self.client_id, self.client_secret,
+                self.redirect_uri, self.scope, test_mode=False)
+        actual = client.get_auth_url(force=True, state='stuff')
+        query = urlencode({
+            'response_type': 'code',
+            'client_id': self.client_id,
+            'redirect_uri': self.redirect_uri,
+            'approval_prompt': 'force',
+            'scope': ' '.join(self.scope),
+            'state': 'stuff'
+        })
+        expected = smartcar.const.CONNECT_URL + '/oauth/authorize?' + query
+        self.assertEqual(actual, expected)
+
+    def test_get_auth_url_test_mode(self):
         oem = 'audi'
         actual = self.client.get_auth_url(force=True, state='stuff')
```

Comment:
Can you create the client here so that this test is easier to understand? It wasn't immediately obvious to me that I had to look at the client shared across these tests.